### PR TITLE
Fixes #16745: when upgrding rudder from 5.0 to 6.0, the rudder-pkg restaure pkg cannot restart correctly Jetty, so rudder-upgrade doesn't checkk compatibility, and rudder cannot start

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/16745

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-webapp-postinst [hunk 1]
```diff
@@ -168,6 +162,13 @@ echo "Done"
 # Adjust permissions on /var/rudder/configuration-repository
 /opt/rudder/bin/rudder-fix-repository-permissions  >> ${LOG_FILE}
 
+# Restore the plugins and checking them
+if [ -f /tmp/rudder-plugins-upgrade ]
+then
+  /opt/rudder/bin/rudder-pkg plugin restore-status < /tmp/rudder-plugins-upgrade
```
Comment (VinceMacBuche) [inline rudder-webapp/SOURCES/rudder-webapp-postinst]: can you replace /opt/rudder/bin/rudder-pkg by 
rudder package

File: rudder-webapp/SOURCES/rudder-webapp-postinst [hunk 2]
```diff
@@ -168,6 +162,13 @@ echo "Done"
 # Adjust permissions on /var/rudder/configuration-repository
 /opt/rudder/bin/rudder-fix-repository-permissions  >> ${LOG_FILE}
 
+# Restore the plugins and checking them
```
Comment (amousset) [inline rudder-webapp/SOURCES/rudder-webapp-postinst]: I dont get this. Currently we restore the status, then rudder-upgrade runs `check-compatibility` and `rudder-postupgrade`. Why doing it again just after?

Comment (ncharles) [reply rudder-webapp/SOURCES/rudder-webapp-postinst]: parent ticket change the order, but it wasn't such a great idea

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by ncharles** [comment]
OK, squash merging this PR

**Pull request merged**